doc/quickstart/vmware_quickstart.rst

```diff
@@ -29,11 +29,11 @@ Software
 ----------------
 Download the VM version of OSGeoLive. You will need the 7-zip compression program to unzip this file. And of course the VMWare Player software for running the virtual machine. Note that VMWare Player, while offered as a cost-free download, is not open source. You must register on the VMWare website, and agree to their licence in order to use the software.:
 
-`The OSGeo virtual machine  <http://download.osgeo.org/livedvd/release/5.5/osgeolive-vm-5.5.7z>`_
+`The OSGeo virtual machine  <https://download.osgeo.org/livedvd/releases/5.5/>`_
```

Comment:
V5.5 is very old, perhaps point to the parent directory (and update the file header above?).

Reply:
@neteler `vmware_quickstart.rst` isn't linked to from anywhere and isn't available at https://live.osgeo.org/en/quickstart/vmware_quickstart.html so I'm thinking this whole file is outdated and could be removed?